src/@typings/cloud-events.ts

```diff
@@ -89,5 +90,6 @@ export interface CloudEvent {
 
 interface Base<D> extends CloudEvent {
   type: CloudEventType;
+  shkeptncontext: string;
```

Comment:
Is this guaranteed to be there? can you like the docs?

Reply:
The Keptn Docs show it for each type, so what going by that as my guide.
https://keptn.sh/docs/0.4.0/reference/custom-service/